ui-v2/src/components/concurrency/task-run-concurrency-limit-empty-state.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from "@/components/ui/button";
+import { DocsLink } from "@/components/ui/docs-link";
+import {
+	EmptyState,
+	EmptyStateActions,
+	EmptyStateDescription,
+	EmptyStateIcon,
+	EmptyStateTitle,
+} from "@/components/ui/empty-state";
+import { PlusIcon } from "lucide-react";
+
+type Props = {
+	onClick: () => void;
+};
+export const TaskRunConcurrencyLimitEmptyState = ({ onClick }: Props) => (
+	<EmptyState>
+		<EmptyStateIcon id="CircleArrowOutUpRight" />
```

Comment:
Random thought: does defining the icon by ID like this affect tree shaking for `react-lucide`?

Reply:
I don't think so: https://lucide.dev/guide/packages/lucide-react#one-generic-icon-component 

I think the next followup PR, we can create a list of Icons only available to the ui using its direct imports.

And then we can create a  Record of _ui-v2's_ Icons and export it that way

Reply:
Gotcha, that sounds like a good way to abstract icon stuff and make it easy to introduce icons from other sources.

Reply:
Addressing in this PR: https://github.com/PrefectHQ/prefect/pull/16208